numpy/lib/arraypad.py

```diff
@@ -539,6 +539,7 @@ def pad(array, pad_width, mode='constant', **kwargs):
         Number of values padded to the edges of each axis.
         ((before_1, after_1), ... (before_N, after_N)) unique pad widths
         for each axis.
+        (before, after) unique pad widths for all axes.
         ((before, after),) yields same before and after pad for each axis.
```

Comment:
The rule is that we effectively broadcast it to `(N, 2)`, everything else just drops out.
But maybe that is too complicated:  What I would suggest is to combine this with the next example though.  It would also be nice to add double backticks ``` `` ``` ariybd akk the tuples to render them like code.

Reply:
I'm sorry which exact example are you suggesting to combine? I'm not sure if I understood this....

Reply:
`(before, after)` and ((before, after),)` do the same thing.

Reply:
ohhhhh make sense now thank you so much!